dev/com.ibm.ws.persistence.ol_fat/publish/features/com.ibm.ws.persistence.consumer-1.0.mf

```diff
@@ -0,0 +1,25 @@
+Subsystem-ManifestVersion: 1
```

Comment:
Since you are not installing this feature as a usr feature, please add the `IBM-Test-Feature: true` manifest attribute so this feature doesn't accidentally get confused with "real" features.

Reply:
Added